website/landing_pages.py

```diff
@@ -0,0 +1,31 @@
+from flask import redirect
+
+from framework.auth import views as auth_views
+from framework.auth.decorators import collect_auth
+
+
+def connected_tools():
+    return _landing_page(title='Connected Tools',
+                         content_path='/public/pages/help/addons.mako',
+                         redirect_to='/settings/addons/')
+
+
+def enriched_profile():
+    return _landing_page(title='Enriched Profile',
+                         content_path='/public/pages/help/user_profile.mako',
+                         redirect_to='/profile/',)
+
+
+@collect_auth
+def _landing_page(*args, **kwargs):
```

Comment:
No problem.

I didn't specify it because I was wrapping `auth_login`, and doing so would mean I couldn't capture positional parameters. Upon consideration, that should never be necessary :)